### ISSUE TITLE
"Numba requires at least version 0.31.0 of llvmlite."

### ISSUE THREAD

**Issue by melrows** [numba/numba#6366]
I am using numba version 0.47.0 in a conda environment for use with Python2.7. 

In python: 
`from numba import jit `
results in
"ImportError: No module named numba"

When I type
`numba -version` 
I get the following error: 
"ImportError: Numba requires at least version 0.31.0 of llvmlite.
Installed version is 0.28.0.
Please update llvmlite."

I have tried 
`conda install -n myenv llvmlite==0.31.0 `
And when I look at 
`conda list`
I see the llvmlite version as 0.31.0, but the "numba -version" command results still shows the same result.

**Comment by esc** [comment]
@melrows thanks for asking about this on the Numba issue tracker. It sounds like your Python installation and/or environment is somehow corrupted. I would suggest setting this up again from scratch.

**Comment by esc** [comment]
@melrows just following up here, did you have any luck re-creating the environments?

**Comment by melrows** [comment]
Partially. I uninstalled and reinstalled conda and still had the same issue. I eventually determined that while conda had the correct version of both numba and llvmlite installed in my conda directory , I had multiple versions of numba and llvmlite installed in my user directory instead of my conda directory. The user directory version was what was called when running numba. I used `which numba` and `which llvmlite` to see which version was being used. I renamed/removed those directories and ran the which codes again to make sure that the version in my conda environment were being run, and was able to successfully run the code.
